xarray/conventions.py

```diff
@@ -508,6 +528,39 @@ def stackable(dim):
                     new_vars[k].encoding["coordinates"] = coord_str
                     del var_attrs["coordinates"]
                     coord_names.update(var_coord_names)
+            for attr_name in CF_RELATED_DATA:
```

Comment:
We should update the docstring for `decode_coords` saying that it controls this behaviour.

Reply:
Somewhat updated.  I'm not sure if I should list the attributes or point to the list of attributes.

Reply:
Let's link to the list if we can.